### DISCUSSION TITLE
Populate with spaces get evaluated Cannot populate path ``  because it is not in your schema. Set the strictPopulate option to false to override.

### DISCUSSION THREAD

**Discussion by masterbater** [Automattic/mongoose#13939] [Q&A] [answered]
Cannot populate path `` because it is not in your schema. Set the `strictPopulate` option to false to override.
If you put two spaces between populate I got the error above
```typescript
 const readings = await Gwapo.find(conditions)
    .populate(
      "user  testgwapo",
      "fname lname isActive height abbrev "
    )
```
`   "mongoose": "^6.12.0",`

**Comment by JeromeD3** [comment]
https://mongoosejs.com/docs/populate.html#query-conditions   you can see this.

**Reply by masterbater** [reply]
if you populate multiple and there is an extra space, you will get that error. This doesnt error in mongoose 5. I already corrected this during migration though

**Reply by vkarpov15** [reply, marked as answer]
This was fixed in #13951
